### ISSUE TITLE
Question on video training RLVR

### ISSUE THREAD

**Issue by pkulium** [thuml/RLVR-World#7]
great work! I have question on steps of video training RLVR? the paper show training loss of 300 steps. what are real training steps/epoch used?

**Comment by Manchery** [comment]
Thanks for your interest in our work!

For our RT-1 experiments, we found that only ~150 steps were needed to achieve substantial improvements, and we report these results in the table.

For our experiments in simulated domains, we found that training needed to run longer (from ~400 to ~700 steps); please see: https://openreview.net/forum?id=jpiSagi8aV
